src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition:
+    transform 300ms ease,
+    background-color 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    font-family: $font-family;
+    color: #060b35;
+    margin-top: 0;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    transition: color 300ms ease;
+
+    .card:hover & {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-family: $font-family;
+    color: #616070;
+    margin-top: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__stars {
+    justify-content: space-between;
+    height: 16px;
+    margin-bottom: 24px;
+    display: flex;
+  }
+
+  &__review {
+    font-family: $font-family;
+    color: #060b35;
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+  }
+
+  &__price {
+    justify-content: space-between;
+    margin-bottom: 16px;
+    display: flex;
+    align-items: center;
+  }
```

Comment:
The `&__price` class should not have `justify-content: space-between;`. This style should be applied in the HTML structure by wrapping price tag and price in a flex container.